other-builtin-functions/05_exploding_arrays/05_exploding_arrays.md

```diff
@@ -0,0 +1,83 @@
+# 04 Expanding arrays into new rows
+
+![Twitter Badge](https://img.shields.io/badge/Flink%20Version-1.3%2B-lightgrey)
+
+> :bulb: This example will show how to create new rows for each element in an array using a `CROSS JOIN UNNEST`.
+
+The source table (`HarryPotter`) is backed by the [`faker` connector](https://flink-packages.org/packages/flink-faker), which continuously generates rows in memory based on Java Faker expressions.
+
+There are many cases where data contains complex data types. Complex data types means that the data is structured in a nested way. Examples of these data types are `ARRAY`, `MAP` or `ROW`.
+
+In this recipe, we'll expand an `ARRAY` into a new row for each element using `CROSS JOIN UNNEST` and join the new rows with a lookup table.
+
+This table DDL creates a `HarryPotter` themed table with three multiple complex data types. It contains a `character_with_age` of data type `MAP<STRING, INT>` that contains three Harry Potter characters and their age. Also included is `spells` of data type `ARRAY` that contain three spells, spoken by them. The last included complex data type is `house_points` of data type ``ROW<`house` STRING, `points` INT>`` that refers to the house of characters and the points given for that spell.
+
+## Script
+
+```sql
+-- Create source table
+CREATE TABLE `HarryPotter` (
```

Comment:
What do you think about making this table simple by removing the MAP and ROW columns? They don't seems to be relevant for the recipe and distract/make it more complicated to follow along.

Reply:
Agreed, I'll use those in different recipes in the future